app.js

```diff
@@ -0,0 +1,49 @@
+
+const express = require('express');
+const hbs     = require('hbs');
+const app     = express();
+const path    = require('path');
+const PunkAPIWrapper = require('punkapi-javascript-wrapper');
+const punkAPI = new PunkAPIWrapper();
+
+app.set('view engine', 'hbs');
+app.set('views', __dirname + '/views');
+app.use(express.static(path.join(__dirname, 'public')));
+app.set("layout", __dirname + "/views/layout.hbs");
+hbs.registerPartials(__dirname + "/views/partials");
+
+app.get('/', (req, res, next) => {
+  res.render('index');
+});
+
+app.get("/beers", (req, res, next) => {
+  // const index = Math.floor(Math.random()*25);
+  // NOT SURE---------------
+  punkAPI.getBeers()
+  .then(beers => {
+    console.log(beers)
+    const chosBeer = beers;
+    res.render("beers.hbs", {chosBeer});
+  })
+  .catch(error => {
+    console.log(error)
+  })
+});
+
+app.get("/random-beers", (req, res, next) => {
+  punkAPI.getRandom()
+  .then(beers => {
+    const chosBeer2 = beers[0];
```

Comment:
you don't have to name it `chosBeer2` since it's in the local scope of this callback, so you can safely name it like the former one